src/applications/vaos/utils/moment-tz.js

```diff
@@ -0,0 +1,1224 @@
+//! moment-timezone.js
```

Comment:
is there a reason to include this lib like this vs. adding it to `package.json` and installing it with Yarn?